File: switch-statement/sw1.cpp
#include<iostream>
using namespace std;
int main(){
    int x=10;
    int y=5;

    switch (x+y){
        case 15:
            cout<<"the result is 15"<<endl;
            break;
        case 20:
            cout<<"the result is 20"<<endl;
            break;
        default:
            cout<<"invalid"<<endl;
    }
    return 0;
}

File: switch-statement/sw1.java
public class sw1 {
    public static void main(String[] args){
        final int x=10; //Using final makes variables constant, preventing changes and improving code safety and readability.
        final int y=10;

        switch (x+y){
            case 15:
                System.out.println("result is 15");
                break;
            case 20:
                System.out.println("result is 20");
                break;
            default:
                System.out.println("invalid");
        }
    }
}
